receiver/prometheusreceiver/config_test.go

```diff
@@ -107,12 +107,10 @@ func TestLoadConfigFailsOnUnknownSection(t *testing.T) {
 
 	factory := NewFactory()
 	factories.Receivers[typeStr] = factory
-	cfg, err := configtest.LoadConfigFile(
-		t,
-		path.Join(".", "testdata", "invalid-config-section.yaml"), factories)
-
+	cfg, err := configtest.LoadConfig(path.Join(".", "testdata", "invalid-config-section.yaml"), factories)
```

Comment:
Should this be LoadConfigAndValidate?

Reply:
I want to ensure that the error is on loading not on validating, probably need to remove the validate check :)